File: HomeLoan/Models/NewApplication.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace HomeLoan.Models
{
    [DataContract]
    public class NewApplication
    {
        [DataMember]
        public loanDetail ld;
        [DataMember]
        public incomeDetail idt;
        [DataMember]
        public personalDetail pd;
        [DataMember]
        public userBankDetail bd;
        [DataMember]
        public document dc;

        NewApplication()
        {
            this.ld = new loanDetail();
            this.idt = new incomeDetail();
            this.bd = new userBankDetail();
            this.pd = new personalDetail();
            this.dc = new document();
        }
    }
}

File: src/app/models/newapplication.ts
import { BankDetail } from './bankDetail';
import { IncomeDetail } from './incomeDetail';
import { LoanDetails } from './loanDetail';
import { Mydocuments } from './mydocuments';
import { PersonalDetails } from './pesonalDetails';

export class NewApplication
{
    ld:LoanDetails;
    idt:IncomeDetail;
    pd:PersonalDetails;
    bd:BankDetail;
    dc:Mydocuments;
    constructor()
    {
        this.ld=new LoanDetails();
        this.idt=new IncomeDetail();
        this.pd=new PersonalDetails();
        this.bd=new BankDetail();
        this.dc=new Mydocuments();
    }
}